CONTRIBUTING_TO_OPENSHIFT_PLUGINS.md

```diff
@@ -0,0 +1,414 @@
+# Development and release process for the OpenShift Jenkins plugins
+
+The OpenShift organization currently maintains several plugins related to integration between V3 OpenShift and V2 Jenkins.
+
+|    Plugin name |  OpenShift Org GitHub URL    |  Jenkins Org GitHub URL    |  Jenkins wiki  |
+| :--------------| --------------------------   | ------------------------   | -------------- |
+| `OpenShift Client Plugin` | [https://github.com/openshift/jenkins-client-plugin](https://github.com/openshift/jenkins-client-plugin) | [https://github.com/jenkinsci/openshift-client-plugin](https://github.com/jenkinsci/openshift-client-plugin) | [https://wiki.jenkins.io/display/JENKINS/OpenShift+Client+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Client+Plugin) |
+| `OpenShift Sync Plugin` | [https://github.com/openshift/jenkins-sync-plugin](https://github.com/openshift/jenkins-sync-plugin) | [https://github.com/jenkinsci/openshift-sync-plugin](https://github.com/jenkinsci/openshift-sync-plugin) | [https://wiki.jenkins.io/display/JENKINS/OpenShift+Sync+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Sync+Plugin) |
+| `OpenShift Login Plugin` | [https://github.com/openshift/jenkins-openshift-login-plugin](https://github.com/openshift/jenkins-openshift-login-plugin) | [https://github.com/jenkinsci/openshift-login-plugin](https://github.com/jenkinsci/openshift-login-plugin) | [https://wiki.jenkins.io/display/JENKINS/OpenShift+Login+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Login+Plugin) |
+| `OpenShift Pipeline Plugin` | [https://github.com/openshift/jenkins-plugin](https://github.com/openshift/jenkins-plugin) | [https://github.com/jenkinsci/openshift-pipeline-plugin](https://github.com/jenkinsci/openshift-pipeline-plugin) | [https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin) |
+
```

Comment:
I had envisioned a more compact table like this: https://markdownshare.com/view/31bb91c3-9da7-4ffc-88f5-3b93e85d8002

Reply:
OK I replaced what you had at that link -thanks